.eslintrc.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
       "react"
   ],
   "rules": {
+    "import/prefer-default-export": "off",
```

Comment:
Yeah I see this rule as having limited value, I'm fine with disabling it. Can you also remove existing overrides in `src/constants/index.js` and `src/utils/test.js`?